src/pages/Home.tsx

```diff
@@ -21,10 +25,137 @@ import {jsx, SxStyleProp} from 'theme-ui';
  * @returns The home page. 
  */
 export const Home: React.FC = () => {
+  //the board
+  const board = useContext<IBoardContext>(BoardContext);
+
+  
+  const [posts, setPosts] = useState<IPost[]>([]);
+
+  
+
+  // TODO: get data from server
+
+  /**
+   * This uses to `fetch` API to request more posts from the server.  
```

Comment:
`the`

Reply:
what do you mean i have perfect grammar 😄